data/nlu/nlu.md

```diff
@@ -1436,6 +1446,8 @@
 - what languages you prefer more speaking at?
 - which languages do you understand?
 - you know [French](language)
+- we cant converse in french?
+- can we converse in french?
```

Comment:
should we annotate these languages? (There might be others missing here as well)

Reply:
hm yeah we probably should.. i can see there's a few in this intent that are unannotated.